Assets/Scripts/UI/CameraIntMsg.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class CameraIntMsg : MonoBehaviour {
+
+    public GameObject Camera;
+    public GameObject ArrowsSide;
+    public GameObject ArrowsUp;
+    public Controller2 isghostMode;
+    private bool onlyOnce;
```

Comment:
I wanted the action of the if statement to only occur once, and in the unity forum it told me to use a boolean and make it true once the if statement is finished, so I used that procedure for it to only occur once.